utils/get.ts

```diff
@@ -0,0 +1,2 @@
+export const get = (obj: any, path: string, defaultValue: any) => path.split('.')
+  .reduce((a, c) => (a && a[c] ? a[c] : (defaultValue || null)), obj);
```

Comment:
Since the function is written so briefly it's not really self documenting. It'd be good to have a jsdoc block above this to explain what it's for, since although we may know it's replacing lodash's `get` no one else would know.

Reply:
👍

Reply:
Added a spec and docs.